setup.py

```diff
@@ -32,10 +34,9 @@
 DOWNLOAD_URL = 'https://github.com/nipy/PySurfer'
 VERSION = version
 
-def check_dependencies():
 
-    needed_deps = ["IPython",
```

Comment:
> The pysurfer command line tool drops you into IPython.

Then it should be an optional dependency, not a hard requirement. People (like me) who do `import surfer` and do everything from there shouldn't need to have IPython.

> Also in general you need IPython to do any interactive work.

I disagree -- I use `python -i` (effectively) from Spyder in general, and rarely (if ever) use IPython, and have no problems with interactivity.